packages/odata-service-writer/src/index.ts

```diff
@@ -91,11 +91,11 @@ async function generate(basePath: string, data: OdataService, fs?: Editor): Prom
     );
 
     const mockserverMiddleware = getMockServerMiddlewareConfig(data);
```

Comment:
Think this loc can be move to here also? https://github.com/SAP/open-ux-tools/pull/61/commits/308c4659c65d02ea5f7a346cfcd022b0e7a408f3#diff-11060fa3e2b31c89f7a31e60dfe2c764b0f9b32db732156920d506581236a4c4R98